src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1809,4 +1809,7 @@ Additional information: {1}</value>
     <value>Default (Open Documents)</value>
     <comment>This text is a menu command</comment>
   </data>
+  <data name="Suppress_hints_when_parameter_names_match_argument_names" xml:space="preserve">
+    <value>Suppress hints when parameter names match argument names</value>
```

Comment:
The phrasing seems all or nothing to me. Perhaps `Suppress hints when parameter name matches argument name`

Reply:
`Suppress hints when argument matches parameter name`